src/Services/UserService.cs

```diff
@@ -36,4 +42,56 @@ public async Task<User> UpdateUserProfileAsync(Guid userId, UpdateUserDTO user)
         
         return updatedUser;
     }
+
+    public async Task<IEnumerable<MonthlySummaryDTO>> GetMonthlySummaryAsync(Guid userId, DateTime startDate, DateTime endDate)
+    {
+        if (userId == Guid.Empty)
+        {
+            throw new ArgumentException("El ID de usuario no puede ser vacío", nameof(userId));
+        }
+
+        if (startDate.Year >= endDate.Year)
+        {
+            throw new ArgumentException("El año de inicio debe ser menor al año de fin", nameof(startDate));
+        }
+
+        if (startDate.Month >= endDate.Month)
+        {
+            throw new ArgumentException("El mes de inicio debe ser menor al mes de fin", nameof(startDate));
```

Comment:
Also with this, keep the code in english

Reply:
Also change the exception to a custom one we have, if the existing execptions does not follow your requirements to throw an error create a new one that suits your error